src/index.js

```diff
@@ -4,11 +4,12 @@
 import { registerPlugin } from '@wordpress/plugins';
 import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
 import { __ } from '@wordpress/i18n';
+import { select, subscribe } from "@wordpress/data";
 
 /**
  * Components
  */
-import CoAuthors from './components/co-authors';
+import CoAuthors from "./components/co-authors";
```

Comment:
Nit: why the `"` double quotes? I see single quotes elsewhere in the code.

Reply:
This was prettify working on its own... I'll manually revert those in another editor